src/registrar/assets/js/get-gov-reports.js

```diff
@@ -54,64 +54,75 @@
 
```

Comment:
Changes to this file are pulled from Zander's unmerged changes from another PR (basically test that this code can be run before running it). This is necessary to avoid a JS error that'll cause an interrupt.